engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/activity/UserTaskFormTest.java

```diff
@@ -27,46 +34,112 @@ public class UserTaskFormTest {
   @ClassRule public static final EngineRule ENGINE = EngineRule.singlePartition();
 
   private static final String PROCESS_ID = "process";
+  private static final String TEST_FORM = "/form/test-form-1.form";
 
   @Rule
   public final RecordingExporterTestWatcher recordingExporterTestWatcher =
       new RecordingExporterTestWatcher();
 
+  @Test
+  public void shouldActivateUserTaskIfFormIsAlreadyDeployed() {
+    // given
+    deployForm();
+    deployProcess("Form_0w7r08e");
+
+    // when
+    final long processInstanceKey = ENGINE.processInstance().ofBpmnProcessId(PROCESS_ID).create();
+
+    // then
+    assertUserTaskActivation(processInstanceKey);
```

Comment:
❌ Let's assert that `JobRecord` contains the `io.camunda.zeebe:formKey` in the custom headers.